packages/amplify-java-template-provider/src/utils/destFileMapper.ts

```diff
@@ -0,0 +1,10 @@
+import _ from 'lodash';
+import path from 'path';
+
+// Converts files to a map of file to destination filename
```

Comment:
update comment. Althought I'm not sure if you should be using this method. I made it to work around the fact that existing node functions nest everything in a 'src' folder but going forward we shouldn't do that.